### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by ekraihan**
Tickets:

#87 
#88 
#89

### CODE REVIEW

File: app/controllers/users.js [hunk 1]
```diff
@@ -50,6 +50,9 @@ export default Ember.Controller.extend(SearchMixin, {
         },
 
         showUserInfo(user) {
+            if(this.get('currentUser'))
```
Comment (ekraihan) [inline app/controllers/users.js]: Fixes issue where edited user data persists

File: app/styles/reports.scss [hunk 2]
```diff
@@ -45,11 +45,26 @@
     left: 0;
 
     .select-id-box {
+        display: flex;
```
Comment (ekraihan) [inline app/styles/reports.scss]: Making id dropdown look nicer

File: app/templates/reports.hbs [hunk 3]
```diff
@@ -6,6 +6,7 @@
                 {{#power-select
                     options = tools
                     selected = selectedId
+                    searchField = id
```
Comment (ekraihan) [inline app/templates/reports.hbs]: HOPEFULLY fix searching issue in tool id dropdown on reports page

File: app/controllers/users.js (hunk 1 above)
Comment (ekraihan) [inline app/controllers/users.js]: Fixes issue where edited user data persists

File: app/templates/reports.hbs (hunk 3 above)
Comment (Jbcampbe) [inline app/templates/reports.hbs]: I think you want to put the `id` in a string... long story short `searchField = id` is treated as `searchField = undefined`

**Pull request merged**